app/worker.php

```diff
@@ -216,17 +211,25 @@
 $platform = new Appwrite();
 $args = $_SERVER['argv'];
 
-if (isset($args[0])) {
-    $workerName = end($args);
-} else {
+if (!isset($args[1])) {
     Console::error('Missing worker name');
+    Console::exit(1);
+}
+
+\array_shift($args);
+$workerName = $args[0];
+$workerIndex = $args[1] ?? '';
+
+if (!empty($workerNum)) {
+    $workerName .= '_' . $workerIndex;
 }
 
 try {
     $platform->init(Service::TYPE_WORKER, [
-        'workersNum' => strtolower($workerName) === 'databases'? 1 :swoole_cpu_num() * intval(App::getEnv('_APP_WORKER_PER_CORE', 6)),
+        'workersNum' => str_starts_with(strtolower($workerName), 'databases') ? 1 : swoole_cpu_num() * intval(App::getEnv('_APP_WORKER_PER_CORE', 6)),
         'connection' => $pools->get('queue')->pop()->getResource(),
         'workerName' => strtolower($workerName) ?? null,
+        'queueName' => App::getEnv('_APP_CONNECTIONS_DB_QUEUE', 'database_db_main')
```

Comment:
Same as above. Let's not use any DB specific default but rather be very generic by adding custom capabilities through a list of env vars that acts as kind of a configuration API.